web/src/click_handlers.js

```diff
@@ -851,7 +851,7 @@ export function initialize() {
     // Workaround for Bootstrap issue #5900, which basically makes dropdowns
     // unclickable on mobile devices.
     // https://github.com/twitter/bootstrap/issues/5900
```

Comment:
Is this comment something we can delete too?

Reply:
Since we are still using `dropdown-menu` for typeaheads, it seems better to have it. It is hard to verify what this fixes.